include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_bwd_weight_two_stage_xdl_cshuffle.hpp

```diff
@@ -452,6 +458,25 @@ struct DeviceGroupedConvBwdWeightTwoStage_Xdl_CShuffle
                             Sequence<CBlockTransferScalarPerVector_NWaveNPerXdl>,
                             I1,
                             I1>;
+    // NPerBlock is used for the first dim which is store dimension
+    // (with CBlockTransferScalarPerVector_NWaveNPerXdl scalar per vector).
```

Comment:
This comment does not tell us much more unless sb is very familiar with `GridwiseElementwise`. So I think you should either elaborate here or write a proper, detailed documentation for `GridwiseElementwise` operation.